src/mme-app/structs.cpp

```diff
@@ -522,3 +522,32 @@ std::ostream& operator << ( std::ostream& os, const DigitRegister15& data_i )
 	os << "\0";	
 	return os;
 }
+
+/*******************************************************
+*BearerCtxtCBResp
+********************************************************/
+BearerCtxtCBResp::BearerCtxtCBResp()
+{
+        memset( &bearer_ctxt_cb_resp_m, 0, sizeof( bearer_ctxt_cb_resp_m ));
+}
+
+BearerCtxtCBResp::BearerCtxtCBResp( bearer_ctxt_cb_resp_t& bearer_ctxt_cb_resp_i )
+{
+        memset( &bearer_ctxt_cb_resp_m, 0, sizeof( bearer_ctxt_cb_resp_m ));
+        memcpy( &bearer_ctxt_cb_resp_m, &bearer_ctxt_cb_resp_i, sizeof( bearer_ctxt_cb_resp_i ));
```

Comment:
is memset really required ? anyway you are doing memcpy immediately on next line.

Reply:
Removed memset.